Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -2024,6 +2024,13 @@ void ACesium3DTileset::Tick(float DeltaTime) {
   glm::dmat4 unrealWorldToCesiumTileset =
       glm::affineInverse(ueTilesetToUeWorld * cesiumTilesetToUeTileset);
 
+  if (glm::isnan(unrealWorldToCesiumTileset[3].x) ||
+      glm::isnan(unrealWorldToCesiumTileset[3].y) ||
+      glm::isnan(unrealWorldToCesiumTileset[3].z)) {
+    // Probably caused by a zero scale.
+    return;
```

Comment:
This takes care of the crash, but we can also validate the scale that causes this a few lines up (line 2020). Maybe even log an error? Might not be obvious to the user why a tileset isn't visible.

Reply:
Logging an error here would spam the output log, because it would happen every frame. We could avoid that, but I'm not sure it's worth the trouble. Hopefully it's not surprising to users that a zero scale means the thing doesn't show up at all?

Reply:
Good point, I didn't think of that. To take it even further, a user might actually be using zero scale to intentionally hide it. If an animation shrinks scale to nothing for example.